packages/@okta/vuepress-site/docs/reference/okta-expression-language/index.md

```diff
@@ -386,7 +386,7 @@ See [Include app-specific information in a custom claim](/docs/guides/customize-
 
 #### Sample using access.scope
 
-In API Access Management custom authorization servers, you can name a claim `scope`. Then, you can use the expression `access.scope` to return an array of granted scope strings.
+In [API Access Management](docs/concepts/api-access-management/) custom authorization servers, you can name a claim `scope`. Then, you can use the expression `access.scope` to return an array of granted scope strings.
```

Comment:
Also a 404 here ... you're missing the leading forward slash before docs (i.e., (/docs/concepts ...)

Reply:
Fixed. Thanks for catching them :)